### PR TITLE
WIP: Translate introductory documentation to Japanese

### PR DESCRIPTION
**Pull request by remoriok**
Adding some introductory documentation (from "About the Documentation" to "Quick Start") into `ja` directory.

An issue is how linking to the missing (non-translated) documentation should be done. I guess the whole documentation need to be published under `ja` directory for navigational consistency, but placing verbatim copies of the original documentation seems not reasonable to me.

Any ideas?

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by remoriok** [comment]
FYI: I have signed the CLA.

File: ja/features.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline ja/features.html]: 2018

**Comment by kkraune** [comment]
Thanks, this is great! Yes, we should not copy the english documents. The best solution would be to link as if the Japanese document exists, and redirect to the English version if it did not. I will merge this so we can start experimenting with the linking.

BTW: I am in Tokyo coming week for the Vespa meetup, we should meet then :-)

**Comment by remoriok** [comment]
Thanks! I'm looking forward to seeing you in Tokyo.

**Pull request merged**